INSTALL.md

```diff
@@ -69,6 +69,24 @@ at [localhost:1081](http://localhost:1081).
 - Keep an eye on console in case of compiling errors.
 - [Read more](https://github.com/Trustroots/trustroots/wiki/Development)
 
+## Enable collecting statistics
+
+* 1. [Install InfluxDB](https://docs.influxdata.com/influxdb/latest/introduction/installation/) 0.9+ and run it (type `influxd`) at default port (8086)
```

Comment:
> Could you add to app init logging a line telling if InfluxDB is configured?

done with a lazy check (on/off based on influxdb.enabled). To check whether everything is running and configured correctly may be quite complex thing to do. Demand it here if necessary (and i'll see what i can do).

> If it's not configured, it should probably console.error only on production env? On test/development it can fail silently.

At the moment there is a check in [messages.server.controller](https://github.com/mrkvon/trustroots/blob/f051c9649412ee69ac72ecd1274e2d6ff21510fd/modules/messages/server/controllers/messages.server.controller.js#L287-L296) before starting stats, so if the influxdb is disabled, no errors are shown. Can be done after this check if required (in the `else`). Other errors are logged later.